src/main/java/com/azure/cosmos/sample/ReadManyItems.java

```diff
@@ -76,178 +79,165 @@ public static void main(String[] args) {
             p.readManyItemsDemo();
             System.out.println("Demo complete, please hold while resources are released");
         } catch (Exception e) {
-            System.out.println("Cosmos getStarted failed with: "+ e);
+            System.out.println("Cosmos getStarted failed with: " + e);
         } finally {
             System.out.println("Closing the client");
             p.close();
         }
         System.exit(0);
     }
-
     // </Main>
 
     private void readManyItemsDemo() throws Exception {
-        System.out.println("Using Azure Cosmos DB endpoint: " +AccountSettings.HOST);
-
-        // Create sync client
-        // <CreateSyncClient>
+        System.out.println("Using Azure Cosmos DB endpoint: " + AccountSettings.HOST);
+        docs = generateDocs(NUMBER_OF_DOCS);
         client = new CosmosClientBuilder()
                 .endpoint(AccountSettings.HOST)
                 .key(AccountSettings.MASTER_KEY)
                 // Setting the preferred location to Cosmos DB Account region
                 // UK South is just an example. User should set preferred location to the Cosmos
                 // DB region closest to the application
                 .preferredRegions(Collections.singletonList("UK South"))
-                .consistencyLevel(ConsistencyLevel.EVENTUAL)
-                .buildClient();
-
-        createDatabaseIfNotExists();
-        createContainerIfNotExists();
+                .consistencyLevel(ConsistencyLevel.SESSION)
+                .buildAsyncClient();
```

Comment:
Since we might be reading the data back after creating the docs, it is important to enable content response on write. 
Example is here - https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples/blob/main/src/main/java/com/azure/cosmos/examples/diagnostics/async/CosmosDiagnosticsQuickStartAsync.java#L73

Reply:
Thanks, done.